lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1509,19 +1509,19 @@ class TestListTopicsAPI(TeamAPITestCase):
     """Test cases for the topic listing endpoint."""
 
     @ddt.data(
-        (None, 401),
-        ('student_inactive', 401),
-        ('student_unenrolled', 403),
-        ('student_enrolled', 200),
-        ('staff', 200),
-        ('course_staff', 200),
-        ('community_ta', 200),
+        (None, 401, None),
+        ('student_inactive', 401, None),
+        ('student_unenrolled', 403, None),
+        ('student_enrolled', 200, 4),
+        ('staff', 200, 6),
+        ('course_staff', 200, 6),
+        ('community_ta', 200, 4),
```

Comment:
I'll have to check the spreadsheet to see if this is correct.

Reply:
![Screen Shot 2020-04-03 at 4 41 27 PM](https://user-images.githubusercontent.com/1639231/78403196-fc4cf680-75c9-11ea-8d13-a0e876d04156.png)

Reply:
Looks right to me